src/main/java/org/arquillian/container/chameleon/ChameleonConfiguration.java

```diff
@@ -141,4 +141,18 @@ private String getOutputDirectory() {
             }
         }
     }
+
+    public static boolean isSupported(String source) throws Exception {
```

Comment:
I wouldn't make it a `static` method and I'm not really sure if that should be a responsibility of `Configuration` class as such. WDYT @lordofthejars